README.md

```diff
@@ -118,8 +119,10 @@ Imagine that you take the code from this question, save it to a file called `nam
 
 What would you have to type to exit the while loop?  Explain your answer.
 
+
```

Comment:
You would need to either type `"SeCrEt" or nothing - an empty string. Either of those inputs would allow us to exit the loop.